test/Common/MoistThermodynamics/runtests.jl

```diff
@@ -404,8 +405,8 @@ end
                 θ_liq_ice,
                 ρ,
                 q_tot,
-                40,
-                FT(1e-3),
+                100,
+                FT(1e-6),
```

Comment:
Seems like unattainable accuracy in practice. Smaller than 1e-2 or maybe 1e-3 does not make sense in practice.

Reply:
Updated this to something more reasonable